### ISSUE TITLE
[Other - Accessibility] Screen reader repetition for icons with headings. (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#55276]
#### VFS team name
CTO 

#### VFS product name
MHV on VA.gov Portal

#### VFS feature name
Authenticated Landing Page

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
As currently coded, each heading is announced by screen readers twice: first announced as the icon, then announced as the heading text itself. This is distracting and since the icons are purely decorative announcing a text alternative for them is not necessary.

### Link, screenshot or steps to recreate
Each icon-plus-heading is currently coded as:
```
<div ...>
     <div ...>
          <div role="img" aria-labelledby="mhv-c-card-appointments" ...></div>
     </div>
     <div ...>
          <h2 ... id="mhv-c-card-appointments">Appointments</h2>
     </div>
</div>
```

When a screen reader encounters an element with a semantic role and an `aria-labelledby`, it announces the text of the element referenced and then proceeds to the next element.

Testing with NVDA, that means this is announced as "Graphic: Appointments, Heading level 2: Appointments."

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
The `aria-labelledby` and `role="img"` aren't necessary here. Since the icon doesn't communicate any meaningful information and is purely decorative, the icon `div` should have `aria-hidden="true"` to hide it from the accessibility tree entirely.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by dcloud** [comment]
Looks like I misread https://design.va.gov/foundation/icons#accessibility-considerations, on the first go-around. It sounds like I _might_ want `role="presentation"` ?

**Comment by briandeconinck** [comment]
Good question! That design system page should be updated. I'll submit a ticket to the DST on that tomorrow.

Since the icons are added using CSS `content: [escaped unicode string]`, I think `aria-hidden` is the safer option. With `role="presentation"`, most modern screen readers will be smart enough to ignore the escaped string. But some screen readers (especially older versions) might make the attempt to read out that unicode string and confuse users. Better to hide it entirely here.

`aria-hidden` is also [recommended by Font Awesome](https://fontawesome.com/v5/docs/web/other-topics/accessibility) for decorative icons.

More generally, @timwright12 wrote a [good blog post](https://timwright.org/blog/2016/11/19/difference-rolepresentation-aria-hiddentrue/) several years ago on the difference between `role="presentation"` and `aria-hidden="true"`.

**Comment by dcloud** [comment]
@briandeconinck is there a way I can/should tag a11y team on the PR for these changes?

**Comment by briandeconinck** [comment]
@dcloud We don't have a formal process for validating that issues have been resolved, it's basically on you to implement fixes and clear it with your PO. But informally, you can just tag me when you've got something you want me to look at. Something small like these changes, I'm happy to just look at a PR in GitHub. If it was a bigger change I might ask for a review instance or something like that.

**Comment by dcloud** [comment]
Okay, I figured it was small enough that our POs checkmark was good, so I merged. But you can see the small diff: https://github.com/department-of-veterans-affairs/vets-website/pull/23660/files
